services/skus/service.go

```diff
@@ -1329,129 +1328,151 @@ func (s *Service) verifyCredential(ctx context.Context, req credential, w http.R
 	logger.Debug().Msg("caveats validated")
 
 	if req.GetType(ctx) == singleUse || req.GetType(ctx) == timeLimitedV2 {
```

Comment:
given we lend both a time-limited and a timeLimitedV2, is it possible to double the subscription "capacity"? Should we be able to disable "time-limited" if timeLimitedV2 is lent once?

Reply:
time-limited credentials are currently strictly used only for brave talk afaik.  time-limited-v2 are "single-use" credentials whereas time-limited are "multiple-use" during the validity period.  The credential type is specified in the products.

Reply:
Hi @thypon, can we resolve this question or is there still anything outstanding? Thanks